static/src/javascripts/projects/common/modules/commercial/ad-prefs.lib.js

```diff
@@ -28,7 +28,7 @@ const setAdConsentState = (provider: AdConsent, state: boolean): void => {
     onConsentSet(provider, state);
 };
 
-const getAdConsentState = (provider: AdConsent): ?boolean => {
+const getAdConsentState = (provider: AdConsent): boolean | null => {
```

Comment:
I thought `?` meant ` | null `.  Is this not allowed in strict flow?

Reply:
`?boolean` is allowed in strict flow, but `?boolean` might be `boolean` or `null`... and it might also mean `undefined`.... yaaaay JavaScript.

I have a minor preference for using this over the maybe type in these kind of cases (where we are returning a value or null), because it makes Flow inference a bit smarter, reduces some boilerplate around refining types, and gives the reader a bit more specific typing.

Other consideration was that the returned value gets used in buildPageTargeting like so:
`adConsentState !== null ? { pa: adConsentState ? 't' : 'f' } : {};`

Which made me a bit 😰 at first since I wondered if we need to account for undefined. 😅